crates/networking/p2p/sync.rs

```diff
@@ -187,12 +187,7 @@ impl SyncManager {
 
         loop {
             debug!("Requesting Block Headers from {search_head}");
-            let block_header_limit = match self.sync_mode {
```

Comment:
per a previous discussion, it seems that when doing snap sync we want to download more headers that when doing full sync?